Description: Integrate free icons from the Tango Project to replace Sun non free icons
--- javahelp2-2.0.05/jhMaster/JavaHelp/src/new/javax/help/SwingHelpUtilities.java	2007-08-16 22:12:18.000000000 +0200
+++ javahelp2-2.0.05.ds1/jhMaster/JavaHelp/src/new/javax/help/SwingHelpUtilities.java	2009-10-19 18:28:00.000000000 +0200
@@ -229,24 +229,24 @@
 	   "HelpContentViewerUI", basicContentViewerUI,
                  "HelpDnDCursor", basicDnDCursor,
 	      "HelpOnItemCursor", basicOnItemCursor,
-               "BackAction.icon", makeBasicIcon("images/Back.gif"),
-            "ForwardAction.icon", makeBasicIcon("images/Forward.gif"),
-              "PrintAction.icon", makeBasicIcon("images/Print.gif"),
-         "PrintSetupAction.icon", makeBasicIcon("images/PrintSetup.gif"),
-             "ReloadAction.icon", makeBasicIcon("images/Reload.gif"),
-          "FavoritesAction.icon", makeBasicIcon("images/Favorites.gif"),
-               "HomeAction.icon", makeBasicIcon("images/Home.gif"),
-      	     "FavoritesNav.icon", makeBasicIcon("images/FavoritesNav.gif"),
-                 "IndexNav.icon", makeBasicIcon("images/IndexNav.gif"),
-                   "TOCNav.icon", makeBasicIcon("images/TOCNav.gif"),
-                "SearchNav.icon", makeBasicIcon("images/SearchNav.gif"),
-              "GlossaryNav.icon", makeBasicIcon("images/GlossaryNav.gif"),
-               "HistoryNav.icon", makeBasicIcon("images/HistoryNav.gif"),
-                "SearchLow.icon", makeBasicIcon("images/SearchLow.gif"),
-             "SearchMedLow.icon", makeBasicIcon("images/SearchMedLow.gif"),
-                "SearchMed.icon", makeBasicIcon("images/SearchMed.gif"),
-            "SearchMedHigh.icon", makeBasicIcon("images/SearchMedHigh.gif"),
-               "SearchHigh.icon", makeBasicIcon("images/SearchHigh.gif")
+               "BackAction.icon", makeBasicIcon("images/Back.png"),
+            "ForwardAction.icon", makeBasicIcon("images/Forward.png"),
+              "PrintAction.icon", makeBasicIcon("images/Print.png"),
+         "PrintSetupAction.icon", makeBasicIcon("images/PrintSetup.png"),
+             "ReloadAction.icon", makeBasicIcon("images/Reload.png"),
+          "FavoritesAction.icon", makeBasicIcon("images/Favorites.png"),
+               "HomeAction.icon", makeBasicIcon("images/Home.png"),
+      	     "FavoritesNav.icon", makeBasicIcon("images/FavoritesNav.png"),
+                 "IndexNav.icon", makeBasicIcon("images/IndexNav.png"),
+                   "TOCNav.icon", makeBasicIcon("images/TOCNav.png"),
+                "SearchNav.icon", makeBasicIcon("images/SearchNav.png"),
+              "GlossaryNav.icon", makeBasicIcon("images/GlossaryNav.png"),
+               "HistoryNav.icon", makeBasicIcon("images/HistoryNav.png"),
+                "SearchLow.icon", makeBasicIcon("images/SearchLow.png"),
+             "SearchMedLow.icon", makeBasicIcon("images/SearchMedLow.png"),
+                "SearchMed.icon", makeBasicIcon("images/SearchMed.png"),
+            "SearchMedHigh.icon", makeBasicIcon("images/SearchMedHigh.png"),
+               "SearchHigh.icon", makeBasicIcon("images/SearchHigh.png")
 	};
 
 	table.putDefaults(uiDefaults);
@@ -302,24 +302,24 @@
 	   "HelpContentViewerUI", basicContentViewerUI,
                  "HelpDnDCursor", gtkDnDCursor,
 	      "HelpOnItemCursor", gtkOnItemCursor,
-               "BackAction.icon", makeGTKIcon("images/Back.png"),
-            "ForwardAction.icon", makeGTKIcon("images/Forward.png"),
-              "PrintAction.icon", makeGTKIcon("images/Print.png"),
-         "PrintSetupAction.icon", makeGTKIcon("images/PrintSetup.png"),
-             "ReloadAction.icon", makeGTKIcon("images/Reload.png"),
-          "FavoritesAction.icon", makeGTKIcon("images/Favorites.png"),
-               "HomeAction.icon", makeGTKIcon("images/Home.png"),
-      	     "FavoritesNav.icon", makeGTKIcon("images/FavoritesNav.png"),
-                 "IndexNav.icon", makeGTKIcon("images/IndexNav.gif"),
-                   "TOCNav.icon", makeGTKIcon("images/TOCNav.gif"),
-                "SearchNav.icon", makeGTKIcon("images/SearchNav.gif"),
-              "GlossaryNav.icon", makeGTKIcon("images/GlossaryNav.gif"),
-               "HistoryNav.icon", makeGTKIcon("images/HistoryNav.gif"),
-                "SearchLow.icon", makeGTKIcon("images/SearchLow.gif"),
-             "SearchMedLow.icon", makeGTKIcon("images/SearchMedLow.gif"),
-                "SearchMed.icon", makeGTKIcon("images/SearchMed.gif"),
-            "SearchMedHigh.icon", makeGTKIcon("images/SearchMedHigh.gif"),
-               "SearchHigh.icon", makeGTKIcon("images/SearchHigh.gif")
+               "BackAction.icon", makeBasicIcon("images/Back.png"),
+            "ForwardAction.icon", makeBasicIcon("images/Forward.png"),
+              "PrintAction.icon", makeBasicIcon("images/Print.png"),
+         "PrintSetupAction.icon", makeBasicIcon("images/PrintSetup.png"),
+             "ReloadAction.icon", makeBasicIcon("images/Reload.png"),
+          "FavoritesAction.icon", makeBasicIcon("images/Favorites.png"),
+               "HomeAction.icon", makeBasicIcon("images/Home.png"),
+      	     "FavoritesNav.icon", makeBasicIcon("images/FavoritesNav.png"),
+                 "IndexNav.icon", makeBasicIcon("images/IndexNav.png"),
+                   "TOCNav.icon", makeBasicIcon("images/TOCNav.png"),
+                "SearchNav.icon", makeBasicIcon("images/SearchNav.png"),
+              "GlossaryNav.icon", makeBasicIcon("images/GlossaryNav.png"),
+               "HistoryNav.icon", makeBasicIcon("images/HistoryNav.png"),
+                "SearchLow.icon", makeBasicIcon("images/SearchLow.png"),
+             "SearchMedLow.icon", makeBasicIcon("images/SearchMedLow.png"),
+                "SearchMed.icon", makeBasicIcon("images/SearchMed.png"),
+            "SearchMedHigh.icon", makeBasicIcon("images/SearchMedHigh.png"),
+               "SearchHigh.icon", makeBasicIcon("images/SearchHigh.png")
 	};
 
 	table.putDefaults(uiDefaults);
